src/OpenRiaServices.Server.EntityFrameworkCore/Framework/EFCoreTypeDescriptionContext.cs

```diff
@@ -44,7 +44,7 @@ public IModel Model
             }
         }
 
-        public IEntityType GetEntityType(Type type) => Model.FindEntityType(type);
+        public IEntityType GetEntityType(Type type) => type?.FullName != null ? Model.FindEntityType(type) : null;
```

Comment:
```suggestion
        // Verify that full name is not null since Model.FindEntityType throws argument exception if full name is null
        public IEntityType GetEntityType(Type type) => type?.FullName != null ? Model.FindEntityType(type) : null;
```